app/controllers/application_controller.rb

```diff
@@ -3,7 +3,18 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
-  def  aaa
-      1 + 1
+  def  aaa (a  ,  b)
+      1 + 1+ 1+ 1+ 1+ 1+ 1+ 1+ 1+ 1+ 1
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Surrounding space missing for operator `+`.